static/bidder-info/cwire.yaml

```diff
@@ -5,6 +5,9 @@ endpointCompression: gzip
 gvlVendorID: 1081
 modifyingVastXmlAllowed: false
 capabilities:
-  site:
+  app:

```

Comment:
Please add an exemplary simple app JSON test.

Reply:
ok!

Reply:
I added two JSONs containing OpenRTB app request and also the additional domainId field.